src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VuapAssertions.java

```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import org.testng.Assert;
+
+import java.util.concurrent.TimeUnit;
+
+public class VuapAssertions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)